cmake/onnxruntime_rocm_hipify.cmake

```diff
@@ -84,6 +84,8 @@ set(contrib_ops_excluded_files
   "transformers/beam_search.h"
   "transformers/greedy_search.cc"
   "transformers/greedy_search.h"
+  "transformers/sampling.cc"
```

Comment:
It seems like sampling.* should already have been excluded from hipification. With changes in this PR, it breaks. So added it to the exclusion list.